content/docs/Quick starts/quick-start-linux.md

```diff
@@ -27,6 +27,7 @@ You must have the following software installed on your machine:
 *   Python version 2.7 or greater
 *   The Python [`pip` command](https://pypi.org/project/pip/)
 *   The Python [virtualenv tool](https://pypi.org/project/virtualenv/)
+*   The ```python-dev``` command 
```

Comment:
I think we should refer to this as a "package". Calling it a command suggests that you can run `python-dev` in a shell to check if it's installed, which is not the case if I understand correctly.

I would also mention that this is named `python-devel` on some Linux distributions.

Reply:
Hi, thanks, I have updated, please verify.